cypress-tests/cypress/e2e/v2/projectBasics.cy.ts

```diff
@@ -0,0 +1,134 @@
+import {
+  getRandomString,
+  validateLoginV2,
+} from "../../support/commands/general";
+import { User } from "../../support/types/user.types";
+import {
+  deleteProjectFromAPIV2,
+  getProjectByNamespaceAPIV2,
+  ProjectIdentifierV2,
+} from "../../support/utils/projectsV2.utils";
+
+const sessionId = ["dashboardV2", getRandomString()];
+
+beforeEach(() => {
+  // Restore the session (login)
+  cy.session(
+    sessionId,
+    () => {
+      cy.robustLogin();
+    },
+    validateLoginV2
+  );
+});
+
+describe("Project - create, edit and delete", () => {
+  // Define some project details
+  const projectNameRandomPart = getRandomString();
+  const projectName = `project/$test-${projectNameRandomPart}`;
+  const projectPath = `project-test-${projectNameRandomPart}`;
+  const projectDescription = "This is a test project from Cypress";
+  const projectIdentifier: ProjectIdentifierV2 = {
+    slug: projectPath,
+    id: null,
+    namespace: null,
+  };
+
+  // Cleanup the project after the test -- useful on failure
+  after(() => {
+    getProjectByNamespaceAPIV2(projectIdentifier).then((response) => {
+      if (response.status === 200) {
+        projectIdentifier.id = response.body.id;
+        projectIdentifier.namespace = response.body.namespace;
+        deleteProjectFromAPIV2(projectIdentifier);
+      }
+    });
+  });
+
+  it("Project - create, edit and delete", () => {
+    // Create a new project
+    cy.visit("/v2");
+    cy.getUserData().then((user: User) => {
+      const username = user.username;
+      cy.get("#plus-dropdown").should("exist").click();
+      cy.getDataCy("navbar-project-new").click();
+      cy.location("hash").should("eq", "#create-project");
+      cy.getDataCy("new-project-modal").should("exist");
+      cy.getDataCy("project-name-input").type(projectName);
+      cy.getDataCy("project-slug-toggle").click();
+      cy.getDataCy("project-slug-input").should("have.value", projectPath);
+      cy.getDataCy("project-visibility-public").click();
+      cy.getDataCy("project-description-input").type(projectDescription);
+      cy.getDataCy("new-project-modal").contains(
+        `The URL for this project will be renkulab.io/v2/projects/${username}/${projectPath}`
```

Comment:
Oh my, oh my, oh my, this is actually hard-coded 🙈 Ouch! 🤕

Reply:
Please remove or comment this test.

Reply:
I am checking to ensure we are showing the correct expected URL.
What would be a reasonable alternative here? Perhaps just `/${username}/${projectPath}`?

Reply:
Oh, true. I made this more resilient to the URL changes and, following your other suggestion on dropping the matcher for `new-project-modal`, I now match the URL element more precisely 2061f524d8cb48cebaf33483b3385e1d6d00be6a